crates/fuzzing/src/oracles/dummy.rs

```diff
@@ -77,8 +77,14 @@ pub fn dummy_memory<T>(store: &mut Store<T>, ty: MemoryType) -> Result<Memory> {
     Memory::new(store, ty)
 }
 
+/// Construct a dummy shared memory for the given memory type.
+pub fn dummy_shared_memory(engine: &Engine, ty: MemoryType) -> Result<SharedMemory> {
+    SharedMemory::new(engine, ty)
+}
```

Comment:
Is this still needed? (or should it perhaps be integrated elsewhere?)

Reply:
Nah, I just removed it